docs/quickstarts/excel-quickstart-vue.md

```diff
@@ -6,7 +6,7 @@ ms.prod: excel
 localization_priority: Priority
 ---
 
-# Build an Excel task pane add-in using Vue
+# Use Vue to build an Excel task pane add-in
```

Comment:
The existing phrasing matches the other quick starts. Let's leave as is unless we plan on changing everything at once.

Reply:
I fixed the other topics as well and committed to this same PR. Better for SEO, accessibility, and more compliant with MSFT style.